index.html

```diff
@@ -232,6 +232,29 @@ <h3>LET'S CONNECT</h3>
         </div>
         <hr class="skills">
     </section>
+    <footer class="footer-section">
+        <div class="contact">
+            <h2>Contact me</h2>
+        </div>
+        <div class="contact-paragraph">
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+        </div>
+        <form action="https://formspree.io/f/mpzbjyrv" method="post">
+            <div class="name">
+                <input name="usr_name" class="name-text" type="text"  placeholder="Frederick" maxlength="30" required>
+            </div>
+            <div class="email">
+                <input name="usr_email" class="email-field" type="email" placeholder="fabienswift@gmail.com" required>
+            </div>
+            <div class="text-field">
+                <textarea class="text-fld" name="text-field" id="text-field" placeholder="Write your message here" maxlength="500" cols="30" rows="10" ></textarea>
+            </div>
+            <div class="footer-btn">
+                <button type="submit" class="bt">Get It Touch</button>
+            </div>
+        </form>
+
+    </footer>
```

Comment:
- [ ] Please adjust the background color of the form to match the Figma design.

![image](https://user-images.githubusercontent.com/62290545/165369317-6e1a1815-8e6f-47d2-80f1-ea6452e84bd0.png)